animal-sniffer/src/main/java/org/codehaus/mojo/animal_sniffer/Clazz.java

```diff
@@ -27,7 +27,7 @@
 
 import java.io.Serializable;
 import java.util.Arrays;
-import java.util.HashSet;
+import java.util.LinkedHashSet;
```

Comment:
Changed all of this to `LinkedHashSet` to make sure the iteration order is preserved.